data/json/items/archery.json

```diff
@@ -400,7 +400,51 @@
       "durability" : 6,
       "clip_size" : 1,
       "reload": 110,
-      "valid_mod_locations": [["underbarrel", 1],[ "sights", 1 ],[ "accessories", 2 ],["stabilizer", 1],["dampening", 1]]
+      "valid_mod_locations": [["underbarrel", 1],[ "sights", 1 ],[ "accessories", 2 ],["stabilizer", 1],["dampening", 1]],
+      "use_action": {
+        "menu_text": "Tighten Limbs",
+        "type": "transform",
+        "target": "compbow_high",
+        "msg": "You tighten the limbs, increasing the strength of the bow."
+      }
+   },
+   {
+      "id" : "compbow_high",
+      "copy-from" : "compbow",
+      "override" : true,
+      "type" : "GUN",
+      "name" : "compound bow (high)",
+      "description" : "A high-power bow with shaped cams and extra cables for high velocity shots that can be used effectively by very strong archers.  Currently set to a high weight, and ready to cause some real damage if you can draw it.",
+      "min_strength" : 8,
+      "dispersion" : 90,
+      "reload" : 120,
+      "ranged_damage" : 22,
+      "range" : 22,
+      "use_action" : {
+        "menu_text" : "Loosen Limbs",
+        "type" : "transform",
+        "target" : "compbow_low",
+        "msg" : "You loosen the limbs, decreasing the strength of the bow."
+      }
+   },
+   {
+      "id" : "compbow_low",
+      "copy-from" : "compbow",
+      "override" : true,
+      "type" : "GUN",
+      "name" : "compound bow (low)",
+      "description" : "A high-power bow with shaped cams and extra cables for high velocity shots that can be used effectively by average archers.  Currently set to a low weight, and much easier to draw.",
+      "min_strength" : 4,
+      "dispersion" : 100,
```

Comment:
`dispersion` already has value of `100` in `compbow`, so it should be removed from here.

Reply:
Fixed